core/internal/gethwrappers/go_generate_test.go

```diff
@@ -206,10 +206,34 @@ func init() {
 }
 
 var (
-	stripWhitespace    = regexp.MustCompile(`\s+`).ReplaceAllString
 	stripTrailingColon = regexp.MustCompile(":$").ReplaceAllString
+	whitespace         = regexp.MustCompile(`\s`).Match
+	quote              = []byte(`"`)[0]
 )
 
+// stripWhitespaceOutsideQuotes removes all whitespace outside " " quotes. Used
+// to normalize JSON for comparison in a hash. Fortunately, no quoted quotes
+// occur in the input
```

Comment:
This feels dangerously hacky, can we not use a JSON library to do this for us?

Reply:
Or `assert.JSONEq`?

Reply:
Currently, the DB tracks hashes of the JSON. Storing the full JSON would be necessary to use `JSONEq`. Essentially, that would be duplicating all the ABI JSON used in `gethwrappers`.

It's hacky, but it's hacky in a way which complains loudly when it breaks: The hashes won't match.

Reply:
You should be using `NormalizedJSON` surely.

Reply:
You're right, I should. Thanks.

Reply:
I can't believe that just worked, even though I'm comparing to `jq` output....